README.md

```diff
@@ -1,16 +1,51 @@
 # Vue Snippets
 
-> A Vue project with simple but nice components.
+Vue projects with simple designs yet powerful features.
 
-## Build Setup
+## Components
 
-``` bash
-# install dependencies
-npm install
+Components i have created:
 
-# serve with hot reload at localhost:8080
-npm run dev
+| Component name | Description                        | Location                  |
+|----------------|------------------------------------|---------------------------|
+| Donation box   | A simple, clean donation component | `src/components/donation` |
```

Comment:
Change location to Path (or namespace?) 
Instead of code, link it like this

| Component name | Description   | Location  |
|----------------|------------------------------------|---------------------------|
| Donation box   | A simple, clean donation component | [src/components/donation](https://github.com/lucasleandro1204/vue-snippets/tree/master/src/components/donation) |